MmSupervisorPkg/Plugin/PostBuildScanMmStandalone_plug_in.json

```diff
@@ -0,0 +1,12 @@
+## @file
+# Build Plugin used to set the path to rc.exe on windows.
+# The plugin is able to use python to locate the tool as to avoid
+# hard-coding the path
+#
+# Copyright (c) Microsoft Corporation.
+# SPDX-License-Identifier: BSD-2-Clause-Patent
+{
+    "scope": "global",

```

Comment:
should this be Windows only given it is using Windows tools

Reply:
Changed to global-win, but it should probably be made its own scope to enable users to only enable this when mm_supv is used.